worlds/ladx/__init__.py

```diff
@@ -102,7 +104,11 @@ class LinksAwakeningWorld(World):
 
     prefill_dungeon_items = None
 
-    player_options = None
+    ladxr_options: LADXRSettings

```

Comment:
Hmmm

You could probably do a quick right-click->refactor on that class and rename it as well, I think that would be good, cause this line causes me a bit of pain

Reply:
I decided to keep it as LADXRSettings and rename the field instead. I'd like to refactor the Settings object in general at some point, but I'm not sure at the moment how tied it is to the upstream randomizer.